README.md

```diff
@@ -1,70 +1,63 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# SpaceX
 
-## Available Scripts
+> A react based application for booking space travels and join missions.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- HTML
+- JavaScript
+- CSS
+- React
+- Redux
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Live Demo Link](https://awaisanwar544.github.io/spacex-project/)
 
-### `npm test`
+## Getting Started
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+**Follow the steps below to run this project on you local machine**
 
-### `npm run build`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+To get a local copy up and running follow these simple example steps.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Prerequisites
+- Understanding of git and github
+### Setup & Install
+- Clone repo on your local device using `git clone <link>` command
+- Run `npm install`
 
```

Comment:
- [ ] Let's try to be detailed in our documentation as it can be of great benefit to non-technical (or new) users. After setting up and installing the packages, users need to start up the app with the `npm start` command. Kindly add it to your README.

- [ ] After you have written your tests. kindly include the command for running the tests: `npm test`